### ISSUE TITLE
Error on Publish required Tag

### ISSUE THREAD

**Issue by SuperSleInt** [netgen/TagsBundle#132]
On Ibexa Experience  3.2.5 and on composer.json :  `"netgen/tagsbundle": "^4.0",`

When I publish an content with required Tag, I have this error : 
```
Uncaught TypeError: Cannot read property 'querySelector' of null
    at EzTagsValidator.validateEzTags (ezplatform-admin-ui-content-edit-parts-js.js:11414)
    at EzTagsValidator.validateField (ezplatform-admin-ui-content-edit-parts-js.js:5904)
    at ezplatform-admin-ui-content-edit-parts-js.js:5984
    at Array.forEach (<anonymous>)
    at EzTagsValidator.isValid (ezplatform-admin-ui-content-edit-parts-js.js:5982)
    at getValidationResults (ezplatform-admin-ui-content-edit-parts-js.js:5506)
    at Array.map (<anonymous>)
    at HTMLButtonElement.clickHandler (ezplatform-admin-ui-content-edit-parts-js.js:5557)
    at HTMLButtonElement.trigger (ezplatform-admin-ui-layout-js.js:1331)
```
And the publication don't work.

But if the Tag is not required, publication works fine.

**Comment by Loobs24** [comment]
I encounter almost the same error on eZ Platform 3.1.5 and netgen/tags-bundle 4.0.4.

When I publish a content with required tag, I have the same behavior as described by @SuperSleInt but with a slighlty different error :
```
Uncaught TypeError: event.target.closest(...) is null
    at validateEzTags /assets/ezplatform/build/ezplatform-admin-ui-content-edit-parts-js.js:11068
    at validateField /assets/ezplatform/build/ezplatform-admin-ui-content-edit-parts-js.js:5872
    at isValid /assets/ezplatform/build/ezplatform-admin-ui-content-edit-parts-js.js:5952
    at isValid /assets/ezplatform/build/ezplatform-admin-ui-content-edit-parts-js.js:5950
    at getValidationResults /assets/ezplatform/build/ezplatform-admin-ui-content-edit-parts-js.js:5508
    at clickHandler /assets/ezplatform/build/ezplatform-admin-ui-content-edit-parts-js.js:5552
    at trigger /assets/ezplatform/build/ezplatform-admin-ui-layout-js.js:1151
```
It seems that this error comes from this pull request : https://github.com/netgen/TagsBundle/pull/124

**Comment by emodric** [comment]
Fixed in 4.0.12
